package.json

```diff
@@ -1,54 +1,54 @@
 {
-  "name": "google-libphonenumber",
-  "version": "3.2.2",
-  "description": "The up-to-date and reliable Google's libphonenumber package for node.js.",
-  "author": {
-    "name": "Rui Marinho",
-    "email": "ruipmarinho@gmail.com"
-  },
-  "license": "(MIT AND Apache-2.0)",
-  "homepage": "https://ruimarinho.github.io/google-libphonenumber/",
-  "bugs": "https://github.com/ruimarinho/google-libphonenumber/issues",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ruimarinho/google-libphonenumber.git"
-  },
-  "main": "dist/libphonenumber.js",
-  "keywords": [
-    "browserify",
-    "e164",
-    "format",
-    "formatting",
-    "international",
-    "libphonenumber",
-    "number",
-    "phone",
-    "phonenumber",
-    "rfc3966",
-    "standardize",
-    "wrapper"
-  ],
-  "options": {
-    "mocha": "--require should test"
-  },
-  "scripts": {
-    "build": "bin/build.sh",
-    "changelog": "github_changelog_generator --no-issues --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,3ba' -e 'P;D' CHANGELOG.md",
-    "lint": "eslint src/index.js test",
-    "prepublish": "npm run build",
-    "test": "npm run build && npm run testonly",
-    "testonly": "mocha $npm_package_options_mocha",
-    "update": "bin/update.sh",
-    "version": "npm run changelog --future-release=$npm_package_version && npm run build && git add -A CHANGELOG.md dist"
-  },
-  "devDependencies": {
-    "browserify": "^13.1.0",
-    "eslint": "^3.5.0",
-    "eslint-config-seegno": "^6.0.0",
-    "mocha": "^3.0.2",
-    "should": "^11.1.0"
-  },
-  "engines": {
-    "node": ">=0.10"
-  }
+    "name": "@remitly/google-libphonenumber",
```

Comment:
I'd actually leave this as `google-libphonenumber` (and not change much of anything else in the package.json) and release tagged versions (like `3.2.2-remitly.1`). That'll make it so narwhal doesn't have to change anything except for upgrade. Hopefully the original source will update at some point and we can stop maintaining a fork.

Reply:
I'll do that with the tag